### PR TITLE
modules/updater: Edited some strings

### PR DESCRIPTION
**Pull request by shxnpie**
Edited some updater strings

### CODE REVIEW

File: friendly-telegram/modules/updater.py [hunk 1]
```diff
@@ -69,9 +69,9 @@ def __init__(self):
 
     async def downloadcmd(self, message):
         """Downloads userbot updates"""
-        await message.edit("Downloading...")
+        await message.edit("Downloading updates...")
         await self.download_common()
-        await message.edit(_("Downloaded! Use <code>.restart</code> to restart."))
+        await message.edit(_("Successfully downloaded updates.\n Please type <code>.restart</code> to restart the bot."))
```
Comment (penn5) [inline friendly-telegram/modules/updater.py]: Please remove the [space] after \n

Comment (penn5) [reply friendly-telegram/modules/updater.py]: Also this line is too long, it must be reduced by one character.

**Comment by shxnpie** [comment]
Ugh my GBoard auto added the space after the .
Also wym by "the line is too long"? I've added the \n for that purpose.

**Comment by penn5** [comment]
@shxnpie The line of code is too long, linter limits it to 120 chars. Any more and lint will be failing, so I won't merge.

**Pull request merged**